### PR TITLE
Fix country include/exclude of nsi #953

### PR DESCRIPTION
**Pull request by frodrigo**

### CODE REVIEW

File: plugins/TagFix_Brand.py [hunk 1]
```diff
@@ -85,9 +88,9 @@ def node(self, data, tags):
                                     tags_to_add[tag] = brands_tags[tag]
                             return {"class": 31301, "subclass": 0,
                                     "fix": {"+": tags_to_add}}
-                        else:
-                            return {"class": 31301, "subclass": 1,
-                                    "fix": {"+": {"brand": tags["name"]}}}
+#                        else:
+#                            return {"class": 31301, "subclass": 1,
+#                                    "fix": {"+": {"brand": tags["name"]}}}
```
Comment (frodrigo) [inline plugins/TagFix_Brand.py:93]: @nlehuby not sure what this part is for. Why just create brand from name when the name is frequent. May be a false positive.

Note: without httis par can also remove the full `_get_frequent_names()`.

Comment (nlehuby) [reply plugins/TagFix_Brand.py:93]: Note sure to understand :thinking:
This part is actually to create brand from name when name is frequent but not part of nsi.
how do you get frequent names if you remove `_get_frequent_names()`

Comment (frodrigo) [reply plugins/TagFix_Brand.py:93]: But why do you want to create brand from frequent name, not using nsi? It the cause of #953

Comment (nlehuby) [reply plugins/TagFix_Brand.py:93]: The purpose was to foster contribution to the nsi: if it's a false positive, a change may be made to remove generic name or add synonyms to existing brand according to the case. If not, the brand metadata may be missing in nsi (and adding a `brand` tag in OSM is still a good addition).

But if you think there are too much false positive, you can remove it and only use nsi and add brand tags to already known brands objects.

Comment (frodrigo) [reply plugins/TagFix_Brand.py:93]: Ok thank you, Yes I prefer remove possible false positive.

**Comment by nlehuby** [comment]
`countryCodes` were remplaced by `locationSet` on last June: https://github.com/osmlab/name-suggestion-index/commit/4323d115472a0c7f44e629c1a3ae54777bbdcc30
I wonder how we can keep track of the format changes in NSI to be more reactive about these...

**Pull request merged**